app/models/setting/provisioning.rb

```diff
@@ -28,7 +28,7 @@ def self.load_defaults
         self.set('libvirt_default_console_address', N_("The IP address that should be used for the console listen address when provisioning new virtual machines via Libvirt"), "0.0.0.0"),
         self.set('update_ip_from_built_request', N_("Foreman will update the host IP with the IP that made the built request"), false),
         self.set('use_shortname_for_vms', N_("Foreman will use the short hostname instead of the FQDN for creating new virtual machines"), false),
-        self.set('websockets_encrypt', N_("Should Foreman encrypt websockets (VNC console access). Choose on, off or auto."), "auto"),
+        self.set('websockets_encrypt', N_("Should Foreman encrypt websockets (VNC/SPICE console access)"), true),
```

Comment:
I don't think encrypted websockets works with SPICE, or at least when this was merged we didn't made it work, [#7018](http://projects.theforeman.org/issues/7018)  is open for that reason.